src/components/Home/index.tsx

```diff
@@ -1,3 +1,18 @@
+import { redditData } from '../../API/contants';
+import API from '../../API';
```

Comment:
name `API` is a little confusing to me so name `axiosInstance` would sound better what do you think.